classes/template/PKPTemplateManager.inc.php

```diff
@@ -1272,6 +1273,34 @@ function smartyAssign($value, $varName, $passThru = false) {
 	}
 
 	/**
+	 * Override the built-in smarty escape modifier to
+	 * add the jqselector escaping method.
+	 */
+	function smartyEscape($string, $esc_type = 'html', $char_set = 'ISO-8859-1') {
+		$pattern = "/(:|\.|\[|\]|,|=|@)/";
+		$replacement = "\\\\\\\\$1";
```

Comment:
backslash needs to be doubled with double quotes.  moreover, with preg_replace, to use backslash in replacement, it must be doubled. So we need \\\\ for a single backslash.